aiidalab/appstore.py

```diff
@@ -115,36 +115,17 @@ def change_vis_list(self, _=None):
 
     def render(self, _=None):
         """Show all the available apps."""
-        max_len = 180
         self.output.clear_output()
         page = self.page_selector.value - 1
         start = page * self.items_per_page.value
         end = (page + 1) * self.items_per_page.value
         with self.output:
-            horisontal_line = ipw.HTML('<hr>')
```

Comment:
I liked the horizontal lines because they help to visually separate apps.

Reply:
Ok.